src/modules/I18n/namespaces.json

```diff
@@ -1,26 +1,16 @@
 [
-  "about",
   "analytics",
-  "case-studies",
   "common",
   "contribute",
```

Comment:
I believe if you do this, i18n will not work for translations within this folder TBD

Reply:
Since `contribute.opentermsarchive.org` and `opentermsarchive.org` are 2 separated projects their is no more `case-studies` folder  in contribute tool.